app/src/main/java/com/appunite/loudius/ui/reviewers/ReviewersScreen.kt

```diff
@@ -61,10 +69,15 @@ fun ReviewersScreen(
 ) {
     val state = viewModel.state
     val snackbarHostState = remember { SnackbarHostState() }
+    val swipeRefreshState = rememberPullRefreshState(
+        refreshing = state.data == Data.Loading,
+        onRefresh = { viewModel.fetchData() },
+    )
 
     ReviewersScreenStateless(
         pullRequestNumber = state.pullRequestNumber,
         data = state.data,
+        pullRefreshState = swipeRefreshState,
```

Comment:
suggestion: maybe we will stick to `pullRefreshState` name instead of `swipeRefreshState` what do you think?